src/main/scala/no/ndla/learningpathapi/controller/LearningpathController.scala

```diff
@@ -543,6 +547,20 @@ trait LearningpathController {
       params.get(paramName).map(_.trim).filterNot(_.isEmpty())
     }
 
+    def paramAsListOfLong(paramName: String)(implicit request: HttpServletRequest): List[Long] = {
+      params.get(paramName) match {
+        case None => List()
+        case Some(param) => {
+          val paramAsListOfStrings = param.split(",").toList.map(_.trim)
+          if (!paramAsListOfStrings.forall(entry => entry.forall(_.isDigit))) {
+            throw new ValidationException(errors = List(ValidationMessage(paramName, s"Invalid value for $paramName. Only (list of) digits are allowed.")))
```

Comment:
Kunne vi kanskje kalt error handleren direkte, i steden for å kaste exceptionen her som scalatra må fange opp for å deretter kalle error handleren. Siden denne funksjonen er i kontroller-klassen bør det være mulig.
Husker vi valgte å gjøre det slik i [article api](https://github.com/NDLANO/article-api/blob/master/src/main/scala/no/ndla/articleapi/controller/ArticleController.scala#L140).